flink/src/main/java/org/apache/iceberg/flink/FlinkCatalogFactory.java

```diff
@@ -52,13 +56,21 @@
 
   // Can not just use "type", it conflicts with CATALOG_TYPE.
   public static final String ICEBERG_CATALOG_TYPE = "catalog-type";
+  public static final String ICEBERG_CATALOG_TYPE_HIVE = "hive";
+  public static final String ICEBERG_CATALOG_TYPE_HADOOP = "hadoop";
+
   public static final String HIVE_URI = "uri";
   public static final String HIVE_CLIENT_POOL_SIZE = "clients";
   public static final String HADOOP_WAREHOUSE_LOCATION = "warehouse";
 
+  public static final String HIVE_SITE_PATH = "hive-site-path";
+  public static final String HIVE_SITE_SCHEMA_FILE = "file";
+  public static final String HIVE_SITE_SCHEMA_HDFS = "hdfs";
```

Comment:
Possible nit / typo / my own confusion on the above two: I think the term is `SCHEME` and not `SCHEMA`, but could be one of those instances where the term is overloaded. I've always heard of that part of the URI referred to as `scheme`.

Reply:
I see in your `getSchema` function that the call to the Hadoop fs API uses `scheme` so I think it is scheme.

```
String schema = path.toUri().getScheme();
```